docs/section22/Rule22-3.md

```diff
@@ -0,0 +1,46 @@
+
+# CHW&CW - Rule 22-3  
+
+**Rule ID:** 22-3  
+**Rule Description:** For Baseline chilled water loop that is not purchased cooling, chilled-water supply temperature shall be reset based on outdoor dry-bulb temperature.  
+**Rule Assertion:** B-RMR = expected value  
+**Appendix G Section:** Section 22 CHW&CW Loop  
+**Appendix G Section Reference:** Section G3.1.3.9 Chilled-water supply temperature reset (System 7, 8, 11, 12 and 13)  
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system that is Type-7, 8, 11, 12, or 13
+2. B-RMR is not modeled with purchased chilled water.
+
+**Manual Check:** None  
+**Evaluation Context:** Building  
+**Data Lookup:** None  
+**Function Call:** None  
+
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system that is Type-7, 8, 11, 12, or 13: `PLACEHOLDER:`
+2. B-RMR is modeled with at least one chilled water system that does not use purchased chilled water: `if B_RMR.ASHRAE229.chillers:`
+
+## Rule Logic:  
+
+- For each chiller in B_RMR: `for chiller_b in B_RMR.ASHRAE229.chillers:`
+
+  - Get cooling loop that chiller servers: `chw_loop_b = chiller_b.cooling_loop`
+
+    - If cooling loop has not been saved in CHW loop array: `if chw_loop_b NOT in chw_loop_array:`
+
+      - Save cooling loop to CHW loop array: `chw_loop_array.append(chw_loop_b)`
```

Comment:
the RDS looks good. I am not sure if we need this chiller check after we have done the applicability check for system 7,8,11,12,13. i.e. once we know that the CHW loop is associated with one of those systems, then we dont need the chiller check. 
we could instead start with identifying zones served by system 7,8,11,12 or 13 and then find the associated fluid loop serving those HVAC systems and then check the reset control. i do not know if one approach is better than tthe other or unnecessarily complicated. would help to know what @mcneillj or @mkarpman think.